db/verify/add-extra-donation.sql

```diff
@@ -0,0 +1,7 @@
+-- Verify magprime:add-extra-donation on pg
+
+BEGIN;
+
+SELECT column_name FROM information_schema.columns WHERE table_name = 'attendee' AND column_name = 'extra_donation';
```

Comment:
Fancy!  We've never written a verification script before, but that looks pretty easy/good.